setup.py

```diff
@@ -56,6 +56,12 @@
           'schema-salad >= 2.6.20170927145003, < 3',
           'typing >= 3.5.3',
           'six >= 1.8.0',
+          'prov == 1.5.1',
+          'pathlib == 1.0.1',
+          'graphviz == 0.8.2',
+          'pygraphviz == 1.3.1',
```

Comment:
Are both needed? Only graphviz seems to be used

Reply:
removed pygraphviz